ArduPlane/tiltrotor.cpp

```diff
@@ -309,8 +309,15 @@ void Tiltrotor::continuous_update(void)
         // Q_TILT_MAX. Anything above 50% throttle gets
         // Q_TILT_MAX. Below 50% throttle we decrease linearly. This
         // relies heavily on Q_VFWD_GAIN being set appropriately.
-       float settilt = constrain_float((SRV_Channels::get_output_scaled(SRV_Channel::k_throttle)-MAX(plane.aparm.throttle_min.get(),0)) * 0.02, 0, 1);
-       slew(MIN(settilt * max_angle_deg * (1/90.0), get_forward_flight_tilt())); 
+       const float throttle = SRV_Channels::get_output_scaled(SRV_Channel::k_throttle);
+       float settilt = 0.0f;
+       if (quadplane.fwd_thr_min < 0) {
+            // Remove throttle min
+            settilt = constrain_float(throttle * 0.02, -1, 1);
+       } else {
+           settilt = constrain_float((throttle - MAX(plane.aparm.throttle_min.get(),0)) * 0.02, 0, 1);
+       }
```

Comment:
```suggestion
       float settilt = 0.0;
       if (quadplane.fwd_thr_min < 0) {
           settilt = constrain_float(throttle * 0.02, -1, 1);
       } else {
            // Remove throttle min
           settilt = constrain_float((throttle - MAX(plane.aparm.throttle_min.get(),0)) * 0.02, 0, 1);
       }
```

Reply:
I wonder what happens all the way at -1. I suspect bad things, we could constrain to the maximum back tilt from the yaw range on vectored yaw tilt rotors. Looks like none vectored tilt-rotors cant do negative tilt at all currently.

Reply:
I don't know if I am tracking fully. I will test out the -1 case though to understand better though

Reply:
With large tilt forward we start changing gains reducing yaw and roll. I'm not sure if that all will still work at large backwards tilt. Not really your problem, just we might expose some new limitations.

Reply:
Aha, ok I see. Happy to test in SITL to see how the gain scaling handles the roll/yaw coupling in backwards tilt angles

Reply:
Haven't seen any issues on the Convergence nor Griffin at steepest back tilts from a controllability perspective. Both are still fairly shallow though (-20 deg)

Reply:
Let me know if there is anything else I can test/improve on this @IamPete1 Appreciate the help!